tools/distpartitioning/gloo_wrapper.py

```diff
@@ -59,6 +59,35 @@ def alltoall_cpu(rank, world_size, output_tensor_list, input_tensor_list):
     for i in range(world_size):
         dist.scatter(output_tensor_list[i], input_tensor_list if i == rank else [], src=i)
 
+def alltoall_cpu_object_lst(rank, world_size, input_list):
+    """
+    Each process scatters list of input objects to all processes in a cluster
+    and return gathered list of objects in output list. 
+    Parameters
+    ----------
+    rank : int
+        The rank of current worker
+    world_size : int
+        The size of the entire
+    output_tensor_list : List of tensor
```

Comment:
output_tensor_list is not in the parameters.

Reply:
Done.